src/screens/Select.js

```diff
@@ -660,6 +663,24 @@ const SelectPage = () => (
           <GenericExtend />
         </Property>
 
+        <Property name="select.emptySearchMessage.container">
+          <Description>
+            Any valid Box prop for the emptySearchMessage container.
```

Comment:
```suggestion
            Any valid Box props for the emptySearchMessage container.
```

Reply:
Not sure if we typically go with singular or plural. I'm fine sticking w convention if we have one.